setup.py

```diff
@@ -332,7 +332,7 @@ def is_Numpy_installed():
     'Bio.Compass',
     'Bio.Crystal',
     'Bio.Data',
-    'Bio.EBI.dbfetch',
+    'Bio.EBI',
```

Comment:
Apologies for the late reply, but I was on holiday.

I've followed your second suggestion.